python/pylibcudf/pylibcudf/io/parquet.pyi

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from pylibcudf.expressions import Expression
+from pylibcudf.io.types import SourceInfo, TableWithMetadata
+
+class ChunkedParquetReader:
+    def __init__(
+        self,
+        source_info: SourceInfo,
+        columns: list[str] | None = None,
+        row_groups: list[list[int]] | None = None,
+        use_pandas_metadata: bool = True,
+        convert_strings_to_categories: bool = False,
```

Comment:
*question*: We don't have filters in the chunked parquet reader, are they supported?

Reply:
For posterity, the answer was yes, the chunked reader does support filter.